spaceship.zsh

```diff
@@ -464,15 +465,21 @@ spaceship_user() {
 # HOST
 # If there is an ssh connections, current machine name.
 spaceship_host() {
-  [[ $SPACESHIP_HOST_SHOW == false ]] && return
+  [[ $SPACESHIP_HOST_SHOW == "never" ]] && return
 
-  [[ -n $SSH_CONNECTION ]] || return
-
-  _prompt_section \
-    "$SPACESHIP_HOST_COLOR" \
-    "$SPACESHIP_HOST_PREFIX" \
-    '%m' \
-    "$SPACESHIP_HOST_SUFFIX"
+  if [[ -n $SSH_CONNECTION ]]; then
+    _prompt_section \
+      "$SPACESHIP_HOST_SSH_COLOR" \
+      "$SPACESHIP_HOST_PREFIX" \
+      '%m' \
+      "$SPACESHIP_HOST_SUFFIX"
+  elif [[ $SPACESHIP_HOST_SHOW == "always" ]]; then
+    _prompt_section \
+      "$SPACESHIP_HOST_COLOR" \
+      "$SPACESHIP_HOST_PREFIX" \
+      '%m' \
```

Comment:
Can we combine this similar to `user` section ? https://github.com/denysdovhan/spaceship-zsh-theme/blob/d39c3545b6a8d8f0284576159dc3b7d362425adf/spaceship.zsh#L450-L454